<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fenetre</class>
 <widget class="QWidget" name="fenetre">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="bImport">
     <property name="text">
      <string>Importer</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="ptCode"/>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Format</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboFormat">
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HTML</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JavaScript (JS)</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>CSS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>JSON</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Output</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combotOutput">
       <item>
        <property name="text">
         <string>Auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Save a File</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Clipboard</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Indentation</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QSpinBox" name="indentationNb">
         <property name="value">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="indentationType">
         <item>
          <property name="text">
           <string>Tab</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Espace</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bCompiler">
       <property name="text">
        <string>Compiler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bDecompiler">
       <property name="text">
        <string>DÃ©compiler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="pbConversion">
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
